config/routes.rb

```diff
@@ -57,4 +59,10 @@
       delete 'clear'
     end
   end
+
+  # @note Only admins should be able to access the Sidekiq web UI.  But this is
+  # accomplished by Puppet configuration restricting access using a shib
```

Comment:
I would say it's accomplished by Apache configuration which is managed in Puppet.